test/e2e/common/storage/volumes.go

```diff
@@ -56,7 +53,6 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-// TODO(#99468): Check if these tests are still needed.
```

Comment:
We have "should store data" tests that run against nfs and gluster:
https://k8s-testgrid.appspot.com/sig-storage-kubernetes#gce&include-filter-by-regex=should%20store%20data&include-filter-by-regex=nfs%7Cgluster

https://github.com/kubernetes/kubernetes/blob/2170bba47a123ce823284d724cf673349c6aa378/test/e2e/storage/testsuites/volumes.go#L156

I think they are pretty similar and we could potentially get rid of the ones under "common/storage/volumes.go". I think the only thing that may be missing is a config for nfs v3. We may have to add another config under https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/in_tree.go#L95

Reply:
Thank you for your reaction!

To verify, you suggest 

* removing the tests from `test/e2e/common/storage/volumes.go`
* And one of two options:
1. Adding a test function with config for nfs v3 at `test/e2e/storage/drivers/in_tree.go` (at the bottom of the file?).
2. Adding a test config for v3 to `SupportedFsType` at `func InitNFSDriver() storageframework.TestDriver` at line 95 from `test/e2e/storage/drivers/in_tree.go` .

Is this correct? If so, I'll directly do this and update the PR.

Reply:
Yeah, I would go with the first approach for adding a nfsv3 driver config